packages/shared/src/libs/types/create-meeting-request-dto.type.ts

```diff
@@ -0,0 +1,6 @@
+type CreateMeetingRequestDto = {
+	meetingLink: string;
+	meetingPassword: string;
+};
```

Comment:
I believe Backend should already have a type in shared folder for creating a meeting